### ISSUE TITLE
External access to port 8812

### ISSUE THREAD

**Issue by bassplay3r** [questdb/questdb-kubernetes#43]
I would like to use a known FQDN for my grafana data source host which uses port 8812  but I'm struggling to see how the chart can support this since the ingress config uses the port defined in the values file for all hosts. If there is a better way please let me know. https://github.com/questdb/questdb-kubernetes/blob/485c4b58281ce308f8976110d904e95368022a86/charts/questdb/templates/ingress.yaml#L38

**Comment by rstreics** [comment]
Hey looking at question and data you have posted it's not clear what are you trying to do.
* 8812 is postgres port and it's will not work in a context of ingress, you need to nodeport for it
* I assume you want to use QuestDB as datasource for grafana?

**Comment by bassplay3r** [comment]
Yes I have a questdb chart that depends on this chart where I override the values. I also have the same for grafana. Each chart is deployed to their own namespace. For now we manually create a postgres data source in grafana and use the CluisterIP of the questdb service and the postgres port.  I tried to use the questdb service name but it did not seem to work. So I'm struggling to understand how I reference the questdb service and postgres port in the grafana data source without having to find the IP address of the service, i.e bake it into the grafana chart. I had initially thought to create an ingress where foo.bar.com points to questdv-service:8812 but maybe that's the wrong approach.

**Comment by rstreics** [comment]
> I tried to use the questdb service name but it did not seem to work.

Did you also specify namespace as well?  "serice_name.namespace".
If this doesn't work I would suggest with basics:
1) create pod for example ubuntu
2) verify that dns work for example via `dig`
3) verify that you connect to that dns and port via `telnet`

**Comment by sklarsa** [comment]
Stale
